### ISSUE TITLE
Getting started link on the main page links to SciPy → is that intended?

### ISSUE THREAD

**Issue by Martin15135215** [hgrecco/pint#1733]
So on the main page https://pint.readthedocs.io/en/stable/ there is the button getting started, and it points to SciPy.

**Comment by keewis** [comment]
thanks for the report. I think this is a duplicate of #1681, and we're only waiting on a release (see also #1731)

**Comment by hgrecco** [comment]
I was thinking that is time for new release. How about this friday? @jules-ch
